src/getty-generator/getty-generator.c

```diff
@@ -11,6 +11,7 @@
 #include "generator.h"
 #include "initrd-util.h"
 #include "log.h"
+#include "macro-fundamental.h"
```

Comment:
This should not be necessary. As it is already indirectly included by many headers.